src/app/core/services/xero/xero-configuration/xero-advanced-settings.service.ts

```
@@ -2,20 +1,128 @@
   providedIn: 'root'
 })
-export class XeroAdvancedSettingsService {
+export class XeroAdvancedSettingsService extends AdvancedSettingsService {
+
+  private apiService = inject(ApiService);
+
+  private workspaceService = inject(WorkspaceService);
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)